core/src/main/java/org/ironjacamar/core/connectionmanager/pool/dflt/DefaultManagedConnectionPool.java

```diff
@@ -594,4 +724,13 @@ public ConnectionListener removeConnectionListener(boolean free)
    {
       return removeConnectionListener(free, listeners);
    }
+
```

Comment:
Remove this method, and use Injection to set the field in the necessary test cases.

lastIdleCheck can be moved to AbstractMCP too